tests/meta/observations/conftest.py

```diff
@@ -73,3 +76,14 @@ def fixture_mip_dna_observations_api(
     mip_dna_observations_api = MipDNAObservationsAPI(cg_config_object, SequencingMethod.WGS)
     mip_dna_observations_api.loqusdb_api = mock_loqusdb_api
     return mip_dna_observations_api
+
+
+@pytest.fixture(scope="function", name="balsamic_observations_api")
```

Comment:
```suggestion
@pytest.fixture(name="balsamic_observations_api")
```
I think default is "function" so it only clutters to be explicit here